libs/design-system/social-media-button/src/lib/social-media-button.component.ts

```diff
@@ -3,7 +3,7 @@ import { ChangeDetectionStrategy, Component, computed, input } from '@angular/co
 import { MatIconModule } from '@angular/material/icon';
 
 /** Social media name type */
-export type SocialMediaName = 'x' | 'facebook' | 'instagram' | 'youtube' | 'linkedin' | 'email';
+export type SocialMediaName = 'x' | 'facebook' | 'instagram' | 'youtube' | 'linkedin' | 'github';
```

Comment:
We still want to have email as a social media button. You'll have to use an if/else block in the html to do it, i.e.
```
@if (name() === 'email') {
  <mat-icon>email</mat-icon>
} @else {
  <mat-icon [svgIcon]="'social:' + name()"?</mat-icon>
}
```